packages/rn-tester/js/examples/TextInput/TextInputExample.ios.js

```diff
@@ -422,6 +422,7 @@ function OnPaste(): React.Node {
           appendLog(JSON.stringify(e.nativeEvent.dataTransfer.types));
           setImageUri(e.nativeEvent.dataTransfer.files[0].uri);
         }}
+        pastedTypes={['fileUrl', 'image', 'string']}
```

Comment:
Could we add a test case where we can _just_ paste a string? And what happens if you try and paste an image?